extensions/wikia/ContributionAppreciation/ContributionAppreciationController.class.php

```diff
@@ -25,6 +25,11 @@ public function appreciate() {
 			// send email that user received appreciation.
 			// Currently disabled because of: https://wikia-inc.atlassian.net/browse/WW-172
 			// $this->sendMail( $revisionId );
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS125)